patches/api/0486-Add-method-to-reset-Lodestone-compasses-back-to-norm.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DerEchtePilz <81232921+DerEchtePilz@users.noreply.github.com>
+Date: Tue, 20 Aug 2024 12:38:20 +0200
+Subject: [PATCH] Add method to reset Lodestone compasses back to normal ones
+
+
+diff --git a/src/main/java/org/bukkit/inventory/meta/CompassMeta.java b/src/main/java/org/bukkit/inventory/meta/CompassMeta.java
+index 5040ab6190b41442986d2a734a8e782df0eab2f6..aae64cafadb1178903f1f054fb0ac2c12f0bdc05 100644
+--- a/src/main/java/org/bukkit/inventory/meta/CompassMeta.java
++++ b/src/main/java/org/bukkit/inventory/meta/CompassMeta.java
+@@ -52,6 +52,15 @@ public interface CompassMeta extends ItemMeta {
+      */
+     void setLodestoneTracked(boolean tracked);
+ 
++    // Paper start - Add method to reset Lodestone compasses back to normal ones
++    /**
++     * If this compass is currently set to track a specific lodestone, this method
++     * will reset this lodestone compass back to a normal compass so that
++     * {@link #isLodestoneTracked()} returns {@code false}.
++     */
++    void resetLodestoneTracked();
```

Comment:
I think the name is now a little wrong. What about `clearLodestone` or something like that? I don't think we need the "tracked" reference anymore.